src/Common/src/System/Security/Cryptography/RSACng.EncryptDecrypt.cs

```diff
@@ -260,7 +260,13 @@ private unsafe byte[] EncryptOrDecrypt(SafeNCryptKeyHandle key, ReadOnlySpan<byt
                 throw errorCode.ToCryptographicException();
             }
 
-            Array.Resize(ref output, numBytesNeeded);
+            if (numBytesNeeded != output.Length)
+            {
+                byte[] ret = output.AsSpan(0, numBytesNeeded).ToArray();
+                CryptographicOperations.ZeroMemory(output);
```

Comment:
should similar clearing happen in the code paths above? (i.e. in the branch on line 242 just before the new allocation)

Reply:
Yeah, it wouldn't hurt.  (It'd be definite yes if we knew that the underlying implementation was doing a partial write, so let's just do it as a defensive thing)